### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by PetrichorPrecipice**
Reorder Glossary

Remove Redundant Content

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -493,17 +493,15 @@ Case 7: Access to Financial Reports
...
-- **Income**: Money received, such as sales revenue.
-- **Expense**: Costs incurred, including staff salaries and product costs.
-- **Mainstream OS**: Windows, Linux, Unix, OS-X
-- **Private contact detail**: A contact detail that is not meant to be shared with others
-- **Transaction**: An exchange of money (e.g. Buying, Selling)
-- **Transactor/Party**: Person that carried out the transaction or is related to the transaction
-- **Income**: Money received (e.g. Sales of product)
-- **Expense**: Costs incurred (e.g. Staff salary, cost of product)
-
+- **Transaction**: An exchange of money (e.g. Buying, Selling).
```
Comment (cxo05) [inline docs/DeveloperGuide.md:503]: Duplicate with line 501?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -493,17 +493,15 @@ Case 7: Access to Financial Reports
 
 ## Glossary
 
+- **Accountant**: A professional who is responsible for managing a company/institution's financial records. 
+- **Expense**: Costs incurred, including staff salaries and product costs.
+- **Mainstream OS**: Windows, Linux, Unix, OS-X.
+- **Private contact detail**: A contact detail that is not meant to be shared with others.
```
Comment (cxo05) [inline docs/DeveloperGuide.md:499]: Dont think we need this

Comment (PetrichorPrecipice) [reply docs/DeveloperGuide.md:499]: Edited.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (PetrichorPrecipice) [reply docs/DeveloperGuide.md:503]: Yep, you're right, removed.

**Comment by cxo05** [comment]
LGTM

**Pull request merged**